clearnode/chart/config/sandbox/secrets.yaml

```
@@ -0,0 +1,9 @@
+config:
+  database:
+    password: ref+tfstategs://terraform-state-deploy/gke-uat-postgresql-admin/default.tfstate/output.postgresql_user_passwords["clearnet_sandbox_admin"]
+  secretEnvs:
+    BROKER_PRIVATE_KEY: ref+gcpsecrets://ynet-stage/clearnet-sandbox-broker-private-key?version=latest
+    POLYGON_AMOI_INFURA_URL: ref+gcpsecrets://ynet-stage/clearnet-sandbox-polygon-amoi-infura-url?version=latest
```

```suggestion
    POLYGON_AMOY_INFURA_URL: ref+gcpsecrets://ynet-stage/clearnet-sandbox-polygon-amoy-infura-url?version=latest
```

And rename the GCP secret and in the Clearnode config as well